include/deal.II/grid/tria.h

```diff
@@ -4946,10 +4946,4 @@ extern template class Triangulation<3, 3>;
 
 DEAL_II_NAMESPACE_CLOSE
 
-// Include tria_accessor.h here, so that it is possible for an end
-// user to use the iterators of Triangulation<dim> directly without
-// the need to include tria_accessor.h separately. (Otherwise the
-// iterators are an 'opaque' or 'incomplete' type.)
-#include <deal.II/grid/tria_accessor.h>
-
```

Comment:
@bangerth This was the main issue. If we are too concerned about this removal, we can turn this header into a meta header and have the `Triangulation` implementation in separate file included from this one.

Reply:
And this one too.